modules/customize-certificates-api-add-named.adoc

```diff
@@ -113,3 +113,8 @@ kube-apiserver   {product-version}.0     True        False         False      14
 ----
 +
 If `PROGRESSING` is showing `True`, wait a few minutes and try again.
+
```

Comment:
Can you please add a "+" on this line, so that the note is indented with step 6?

```suggestion
+
```